contents/careers.md

```diff
@@ -83,6 +99,14 @@ This way, you'll get a chance to see how we work, and we'll get a chance to see
 
 # Open Roles
 
+## Unspecified
```

Comment:
Unspecified sounds weird. If we think 'Open Application' (the usual term for this) is bad, we should think of something else.

Reply:
Fixed.